src/sickle.ts

```diff
@@ -36,12 +43,15 @@ class Annotator {
 
   constructor(private options: SickleOptions) { this.indent = 0; }
 
-  annotate(file: ts.SourceFile): string {
+  annotate(file: ts.SourceFile): SickleOutput {
     this.output = [];
     this.file = file;
     this.visit(file);
     this.assert(this.indent == 0, 'visit() failed to track nesting');
-    return this.output.join('');
+    return {
+      output: this.output.join(''),
+      externs: null,
+    };
```

Comment:
In general, do you prefer this pattern "interface + object literal", vs "class + new constructor" for simple objects where the differences don't matter? I don't have much of an opinion either way, but we should be consistent in the sickle codebase.